compiler/AST/AggregateType.cpp

```diff
@@ -986,6 +986,22 @@ AggregateType* AggregateType::generateType(CallExpr* call,
 
   INT_ASSERT(notNamed.size() == 0);
 
+  // Don't even allow creating an instantiation of something like
+  // 'owned' with a non-class.
+  bool isManagedPtrType = symbol->hasFlag(FLAG_MANAGED_POINTER);
+  if (isManagedPtrType && genericFields.size() > 0) {
+    // Assume first generic field is the underlying type.
+    auto field = genericFields[0];
+
+    if (auto sub = toTypeSymbol(map.get(field))) {
+      if (!isClassLike(sub->typeInfo())) {
+        USR_FATAL(call, "cannot use type non-class type '%s' with memory management specifiers",
```

Comment:
the double usage of "type" doesn't parse right.

I suggest just removing the first "type"

Reply:
Yeah I just noticed that one.